seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-1/src/test/resources/jdbc_mariadb_source_and_sink.conf

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  jdbc {
+    url = "jdbc:mariadb://mariadb-e2e:3306/seatunnel"
+    driver = "org.mariadb.jdbc.Driver"
+    connection_check_timeout_sec = 100
+    user = "mariadb_user"
+    password = "mariadb_password"
+
+    query = "select * from source;"
```

Comment:
I think the generic dialect should support `table_path` and catalog table too.

Reply:
oh. It aleady supported by `JdbcCatalogUtils`.

Reply:
@Hisoka-X done